packages/core/src/lib/actions/base.action.ts

```diff
@@ -1,12 +1 @@
-export type BaseActionState = 'started' | 'firing' | 'stopped';
-export class BaseAction {
-  mouseX: number;
-  mouseY: number;
-  ms: number;
-
-  constructor(mouseX: number, mouseY: number) {
-    this.mouseX = mouseX;
-    this.mouseY = mouseY;
-    this.ms = new Date().getTime();
-  }
-}
+export class BaseAction { }
```

Comment:
no reason to keep an empty class and extend it, I think we can instead take the similarities between the different Actions (Mouse, Keyboard, etc..), reduce the shared logic into the BaseAction and then extend it